SWLOR.Game.Server/Feature/AbilityDefinition/Armor/ProvokeAbilityDefinition.cs

```diff
@@ -80,7 +80,7 @@ private void Provoke2()
                         if (GetDistanceBetweenLocations(GetLocation(nearest), location) > 8f)
                             break;
 
-                        if (!GetIsPC(nearest))
+                        if (!GetIsPC(nearest) && BeastMastery.IsPlayerBeast(activator) == false)
```

Comment:
Do a henchman check here instead. This won't work for droids as written. GetAssociate is the function you need.

Reply:
Also, I think this should be caught in the enmity service rather than here. Otherwise every place we adjust enmity could run the risk of including henchmen.